### PR TITLE
Add README.md and a Code Hunt vote badge

### PR DESCRIPTION
**Pull request by CodeHuntIO**
@LeaVerou FYI, your repo has been featured today on @CodeHuntIO   

http://codehunt.io/sub/stretchy/

**TL;DR** Code Hunt helps programmers discover awesome code — like yours!   

This pull-request adds a "vote" badge to your README.md:   

[CodeHunt.io](http://codehunt.io/sub/stretchy/?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)   

Fingers crossed that we can drive more attention to your brilliant work.   

Happy coding!
Chris ^_^   

p.s. I hope you found this PR helpful, if not please [let me know](mailto:chris@codehunt.io).   

p.p.s I've also tweeted about you here https://twitter.com/hunt_css/status/626726717315543040  

p.p.p.s Stretchy is the first "CSS" post on Code Hunt and I was super nervous sending this to you because I have a ton of respect for your work :flushed: I hope that adding this README to your repo helps other devs and please feel free to remove the codehunt badge if you don't want to "endorse-by-association" something so new/unheard-of. Cheers!

### CODE REVIEW

**Comment by LeaVerou** [comment]
Hi Chris!

Sorry for the huge delay!
Thanks for the contribution, but I try to avoid duplicating the docs in the README because one of them is bound to get out of sync. 
Nice work on codehunt.io, I think I'll subscribe! :)

**Comment by LeaVerou** [comment]
That said, if you want to send a PR about a README that doesn't duplicate info from the website, you are very welcome :)

**Comment by CodeHuntIO** [comment]
Cheers Lea :-)
